cms/templates/cms/home_page.html

```diff
@@ -70,7 +70,11 @@
               {% for program in programs %}
               <li class="col-sm-6">
                 {% if program.programpage %}
-                  <a href="{{ program.programpage.url }}" class="program-link">
+                	{% if program.programpage.external_program_page_url %}
+                	<a href="{{ program.programpage.external_program_page_url }}" class="program-link" target="_blank">
```

Comment:
This should not have the `target="_blank"` according to [this recent comment](https://github.com/mitodl/micromasters/issues/598#issuecomment-227875534)